tasks/main.yml

```diff
@@ -28,7 +28,13 @@
     --os-interface=public
     cluster-template-create
     {% for name, value in item.item.items() %}
+      {% if name == 'flags' %}
+        {% for flag in value %}
+    --{{ flag | replace('_', '-') }}
+        {% endfor %}
+      {% else %}
     --{{ name | replace('_', '-') }}='{{ value }}'
+      {% endif %}
```

Comment:
Any reason not to use an empty value to represent a flag?

Reply:
would that work?

Reply:
wouldn't that resolve to `--floating-ip-disabled=` which is a syntax error?

Reply:
Yes, you'd need to make a change to support it, e.g.

`{% if value is not none %}'{{ value }}'{% endif %}`

Reply:
oh. does this offer an advantage over iterating over a list of flags? would be more confusing for the reader to have empty variables no?

Reply:
I guess it's personal preference. Either way it needs documenting.